test/fixtures/globals.ts

```diff
@@ -14,3 +14,4 @@ const {window} = new JSDOM('…');
 (global as any).URL = window.URL;
 
 document.head.insertAdjacentHTML('beforeend', '<link href="https://github.com/avajs/ava/commits/master.atom" rel="alternate" title="Recent Commits to ava:master" type="application/atom+xml">');
+document.head.insertAdjacentHTML('beforeend', '<a hidden="" data-hotkey="t" data-pjax="true" href="https://github.com/avajs/ava/find/master"></a>');
```

Comment:
Does any test currently fail without this line? If not, maybe a new test should be added

Reply:
I do not think its needed any more. However leave the test for now I will come back to it later. I need to fully test this PR first. (later tonight)